Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -25,21 +25,10 @@ private enum SearchListSection: Int{
     }
 }
 
-// A delegate to support clicking on rows in the view controller
-// This needs to be accessible to objc for UIViewControllers to call it
-@objc
-protocol SearchViewControllerDelegate: class {
-    func searchViewController(searchViewController: SearchViewController, didSubmitURL url: NSURL)
-}
-
-class SearchViewController: UIViewController {
-    weak var delegate: SearchViewControllerDelegate?
-    private var tableView = UITableView()
+class SearchViewController: HomePanelTableViewController {
```

Comment:
Eh, this doesn't really make sense since SearchViewController isn't a HomePanel. In fact, HomePanelController is completely decoupled from SearchViewController: SearchViewController is created and owned by BVC (and it's is in <b>Frontend/Browser</b>, not <b>Frontend/Home</b>).

Options I can think of:
  1. Rename HomePanelTableViewController to SiteTableViewController (or something similar) so that it's not specific to home panels.
  1. Don't have SearchViewController extend HomePanelTableViewController.
  1. Couple SearchViewController to home (although it still technically wouldn't be a panel).

I think I'd prefer them in that order, but your call. I wouldn't block on this, but please file a follow-up to clean this up if you don't want to do it here.